dev/a11y_assessments/lib/use_cases/material_banner.dart

```diff
@@ -15,57 +15,44 @@ class MaterialBannerUseCase extends UseCase {
   String get route => '/material_banner';
 
   @override
-  Widget build(BuildContext context) => const MainWidget();
+  Widget build(BuildContext context) => MainWidget();
 }
 
-class MainWidget extends StatefulWidget {
-  const MainWidget({super.key});
+class MainWidget extends StatelessWidget {
+  MainWidget({super.key});
 
-  @override
-  State<MainWidget> createState() => MainWidgetState();
-}
-
-class MainWidgetState extends State<MainWidget> {
-  double currentSliderValue = 20;
-  ScaffoldFeatureController<MaterialBanner, MaterialBannerClosedReason>? controller;
+  final FocusNode dismissButtonFocusNode = FocusNode();
```

Comment:
This focus node is never disposed. You'll have to keep the `MainWidget` stateful so you can dispose the focus node in the dispose method of that widget's State class.

You can also keep the `MainWidget` constructor const.

Reply:
Thanks for the review! Added the state back to the widget along adding another focus node to put focus back on the show button whenever dismiss is pressed.

However, I'm running into a weird issue that I can't figure out:
[Screen recording 2024-08-02 1.52.03 PM.webm](https://github.com/user-attachments/assets/0125b4d3-cb19-408d-985e-af665e2aea05)

When clicking the show button and dismiss button, the behavior works fine, but whenever I press Enter on the dismiss button, I keep getting an assertion error with InkWell saying "InkResponse must be done with layout before starting a splash.". This only happens when pressing Enter on the dismiss button, and I can't quite figure out what the issue is. Any suggestions would be greatly appreciated!